Bot crashes from any input

**Describe the bug**
After running and installing all the dependencies, the bot gives an error and crashes. This is both with the docker install, as well as the yarn install.

```[Nest] 2006280  - 01/21/2023, 7:26:04 PM   DEBUG [JellyfinWebSocketService] Opening WebSocket with client id jellyfin-discord-bot...
node:events:368
      throw er; // Unhandled 'error' event
      ^

Error: Unexpected server response: 404
    at ClientRequest.<anonymous> (/root/jellyfin-discord-music-bot/node_modules/ws/lib/websocket.js:888:7)
    at ClientRequest.emit (node:events:390:28)
    at HTTPParser.parserOnIncomingClient (node:_http_client:623:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:128:17)
    at TLSSocket.socketOnData (node:_http_client:487:22)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:199:23)
Emitted 'error' event on WebSocket instance at:
    at emitErrorAndClose (/root/jellyfin-discord-music-bot/node_modules/ws/lib/websocket.js:1004:13)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
error Command failed with exit code 1.```

The docker-compose still doesn't work, the bot gets stuck on thinking after issuing a command, and the songs don't show up as I am searching them. nothing relevant in the logs either.

Also tried grabbing the lastest source and compiling and I get this error on yarn build:
```
src/commands/play/play.comands.ts:159:42 - error TS2345: Argument of type 'string | SearchType | undefined' is not assignable to parameter of type 'SearchType | undefined'.
  Type 'string' is not assignable to type 'SearchType | undefined'.

159       PlayCommandParams.getBaseItemKinds(type),
                                             ~~~~

Found 1 error(s).

error Command failed with exit code 1.
```

I feel its just my machine messing up, I'm going to spin up another remote server and see if i get the same issues

--- a/src/clients/jellyfin/jellyfin.stream.builder.service.ts
+++ b/src/clients/jellyfin/jellyfin.stream.builder.service.ts
@@ -18,7 +18,7 @@ export class JellyfinStreamBuilderService {
     const accessToken = this.jellyfinService.getApi().accessToken;
 
     const uri = new URL(api.basePath);
-    uri.pathname = `/Audio/${jellyfinItemId}/universal`;
+    uri.pathname += `/Audio/${jellyfinItemId}/universal`;
     uri.searchParams.set('UserId', this.jellyfinService.getUserId());
     uri.searchParams.set(
       'DeviceId',

--- a/src/clients/jellyfin/jellyfin.websocket.service.ts
+++ b/src/clients/jellyfin/jellyfin.websocket.service.ts
@@ -154,7 +154,7 @@ export class JellyfinWebSocketService implements OnModuleDestroy {
 
   private buildSocketUrl(baseName: string, apiToken: string, device: string) {
     const url = new URL(baseName);
-    url.pathname = '/socket';
+    url.pathname += '/socket';
     url.protocol = url.protocol.replace('http', 'ws');
     url.search = `?api_key=${apiToken}&deviceId=${device}`;
     return url;

